src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateGcsBucketStep.java

```diff
@@ -53,6 +59,14 @@ public StepResult doStep(FlightContext flightContext)
     FlightMap inputMap = flightContext.getInputParameters();
     ApiGcsBucketCreationParameters creationParameters =
         inputMap.get(CREATION_PARAMETERS, ApiGcsBucketCreationParameters.class);
+    String gcpProjectId =
```

Comment:
It doesn't hurt to have this here, but the check has already been done in the metadata step in ResourceDao around line 240.

Reply:
Yep - I had to pull the value of of optional here and it seemed useful to have an informative error message, but this could also be a "this should never happen" error.

Reply:
It is fine. If someone goes are re-orders the steps, the check will still be enforced.